client/attest.go

```diff
@@ -320,6 +321,8 @@ func getTEEAttestationReport(attestation *pb.Attestation, opts AttestOpts) error
 	}
 
 	// Try SEV-SNP.
+	// TODO this is not working, even on a non SNP machine, CreateSevSnpQuoteProvider
```

Comment:
Eh, the previous go-sev-guest version only checks if the vm has a configfstsm or not. It does not check if it is a SNP machine. 

With the https://github.com/google/go-sev-guest/commit/de2ac900e958c59c98a9573b6a3dc2b9b26b5156 being merged, CreateSevSnpQuoteProvider should returns an error on a non-snp machine.

Reply:
go to know, updated the dependency